sklearn/neighbors/regression.py

```diff
@@ -122,8 +122,8 @@ def predict(self, X):
 
         Parameters
         ----------
-        X : array or matrix, shape = [n_samples, n_features]
-
+        X : array of shape [n_samples, n_features]
```

Comment:
I think we don't usually have an "of" but a ``shape (n_samples, n_features)``. Feel free to ignore though ;)